ksqldb-functional-tests/src/test/resources/query-validation-tests/filter.json

```diff
@@ -0,0 +1,100 @@
+{
+  "comments": [
+    "Tests covering the filter lambda."
+  ],
+  "tests": [
```

Comment:
Can we have all the test records have different keys? Makes things easier to keep track of which output record belongs to which input record when debugging